lib/perl/Genome/Model/Build/ReferenceSequence/Converter.pm

```diff
@@ -57,11 +57,20 @@ sub __errors__ {
     return @errors;
 }
 
+sub exists_for_references {
+    my $class = shift;
+    my $source_reference = shift;
+    my $destination_reference = shift;
+
+    my @ref = $class->_faster_get(source_reference_build => $source_reference, destination_reference_build => $destination_reference);
```

Comment:
Cheating. Reference sequence converters are implemented as SRs but don't hold data and thus aren't useful to track with sponsors and requestors.  `_faster_get()` allows us to bypass the mandatory user relationships.  It's all contained within the class--calling `_faster_get` from the outside would be bad practice.